embeddings/pipeline/hps_pipeline.py

```diff
@@ -97,20 +106,14 @@ def get_best_paramaters(self, study: Study) -> Metadata:
         parsed_params = self.config_space.parse_parameters(dict(best_params, **constant_params))
         return self._get_metadata(parsed_params)
 
-    def run(
-        self,
-    ) -> Tuple[pd.DataFrame, Metadata]:
+    def run(self, run_name: Optional[str] = None, **kwargs: Any) -> Tuple[pd.DataFrame, Metadata]:
         self._pre_run_hook()
         if self.preprocessing_pipeline is not None:
             self.preprocessing_pipeline.run()
         study: Study = optuna.create_study(
-            direction="maximize",
-            sampler=self.sampler,
-            pruner=self.pruner,
-        )
-        study.optimize(
-            self.objective, n_trials=self.n_trials, show_progress_bar=True, catch=(Exception,)
+            direction="maximize", sampler=self.sampler, pruner=self.pruner, study_name=run_name
         )
+        study.optimize(self.objective, n_trials=self.n_trials, show_progress_bar=True)
```

Comment:
should `catch=(Exception,)` be removed in these lines?

Reply:
fixed, now it can be parametrized. By default, it does not catch any exception, which is helpful during the development.